website/content/docs/lambda/registration.mdx

```diff
@@ -0,0 +1,271 @@
+runs as a Lambda function that is invoked by EventBridge in two ways:
+- [CloudTrail Lambda
+  events](https://docs.aws.amazon.com/lambda/latest/dg/logging-using-cloudtrail.html) - This syncs updates and deletes from Lambda into Consul in real-time.
+    Typically, Lambda registrator syncs Lambdas into Consul within one minute,
+    but CloudTrail events are occasionally delayed.
+- [Schedules](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-create-rule-schedule.html) - This periodically does a full sync between Lambda and Consul to prevent
+  entropy. By default, EventBridge triggers a full sync every five minutes.
+
+
+### Architecture
+
+This diagram shows the flow of events from EventBridge into Consul:
+
+<ImageConfig width={500}>
+
+![Lambda Registrator Architecture](/img/lambda_registrator_architecture.svg)
+
+</ImageConfig>
+
+1. EventBridge invokes Lambda registrator. This is either done based on
+   CloudTrail Lambda events or a schedule.
+2. Lambda registrator determines how to reconcile Lambda's control plane state
+   with Consul state and ensures they are in sync by registering, updating and
+   deregistering Lambda services.
+
+### Deploy Lambda Registrator
+
+#### Deploy Lambda Registrator
+
+Here is an example usage of the Lambda registrator terraform module:
+```hcl
+module "lambda-registrator" {
+  source  = "hashicorp/consul-lambda-registrator/aws//modules/lambda-registrator"
+  name = "consul-lambda-registrator"
+  consul_http_addr = "https://consul.example.com:8501"
+  ca_cert_path = aws_ssm_parameter.ca-cert.name
+  http_token_path = aws_ssm_parameter.acl-token.name
+}
+```
+
+Deploy Lambda registrator with `terraform apply`.
+
+#### Optional: Store the CA Certificate in Parameter Store
+
+When Consul making request to the [HTTP API](/api-docs) over HTTPS, Lambda
+registrator uses a CA Certificate stored in Parameter Store. Consul’s Server CA
+can be stored into Parameter Store by applying the following Terraform:
+
+```hcl
+resource "aws_ssm_parameter" "ca-cert" {
+  name        = "/lambda-registrator/ca-cert"
+  type        = "SecureString"
+  value       = <VALUE>
+}
+```
+
+#### Optional: Store the ACL Token in Parameter Store
+
+When Consul has ACLs enabled, Lambda registrator uses an ACL token stored in
+Parameter Store. This can be done with the CLI, API, or Terraform provider.
+With the CLI, it is done by:
+1. Write the following to rules.hcl:
+```hcl
+service_prefix "" {
+  policy = "write"
+}
+```
+
+2. Create the policy with `consul acl policy create -name "lambda-registrator-policy" -rules @rules.hcl`.
+3. Create the token with `consul acl token create -policy-name "lambda-registrator-policy"`.
+4. Store the token in Parameter Store by applying the following Terraform:
+```hcl
+resource "aws_ssm_parameter" "acl-token" {
+  name        = "/lambda-registrator/acl-token"
+  type        = "SecureString"
+  value       = <VALUE>
+}
+```
+
+#### Configuration Options
+
+| Name | Type |  Description |
```

Comment:
```suggestion
| Name | Type |  Description |
```
I'm not too clear on what the `Type` column refers to. I would expect the contents of this column to be something like `string` or `boolean`. Does `Implementation` make more sense for the column header? Would it be more valuable to keep it as `Type` and change the contents of the table?

Reply:
I removed the type column. I agree it doesn't add very much value.